docs/DEPLOYMENT.md

```diff
@@ -42,14 +49,22 @@ stringData:
   REFRESH_JWT_EXPIRES_IN: time such as 15m
 ```
 
-**NOTE:** DO NOT commit and push
+#### Config File
+
+If you want to change the config file passed to hub, you can edit the `30-api-configmap.yaml` and change the URL to the file. By default it is pointing to [config.yaml][config.yaml] in Hub which has Application Data.
```

Comment:
Could you please provide the link to `30-api-configmap.yaml` and a code block of where the user needs to edit

Reply:
sure